### DISCUSSION TITLE
Receiving update/signal when user clicks on waveform canvas to change the play-head

### DISCUSSION THREAD

**Discussion by Megidd** [naomiaro/waveform-playlist#179] [Q&A] [answered]
Is it possible to receive a signal when user clicks on waveform canvas? For example, here I click on waveform canvas to change the location of *black* vertical line. So, the audio is replayed from my click location:

![image](https://user-images.githubusercontent.com/17475482/144848719-3248d9de-c366-4300-ad1c-a08d6b7e6b77.png)

Is there something like this:

```js
var ee = playlist.getEventEmitter();

ee.on("clickupdate", (time) => {
    // User clicks on a location on waveform canvas equivalent to time.
    // Do something with time.
});
```

**Comment by Megidd** [comment, marked as answer]
Well, sorry. I guess I figured it out:

```js
ee.on("select", (start, end) => {
    // `start` is actually the click location in seconds!
    // Do something with `start`. 
});
```

Is it the right approach?

**Reply by naomiaro** [reply]
Yep! That's the right approach. `end` will be different than `start` if the playlist is in select mode and a user has clicked and dragged a selection area.
